WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -436,57 +428,56 @@ local methods = {
             end
         end
 
-        function self.callbacks.OnDragStart(_, _)
+        function self.callbacks.OnDragStart(frame, _)
             if WeakAuras.IsImporting() or not data.parent then return end;
 
-            data.dragging = true
+            local x, y = GetCursorPosition()
+            local scale = UIParent:GetScale()
 
-            self.frame:StartMoving()
-            self.frame:SetUserPlaced(false)
+            frame.isDragged = true
+            frame:StartMoving()
+            frame:ClearAllPoints()
+            frame:SetPoint("Center", UIParent, "BOTTOMLEFT", x/scale, y/scale)
+
+            WeakAuras:SortDisplayButtons() -- find a better way to trigger a DoLayout call on the scroll frame
         end
 
-        function self.callbacks.OnDragStop(_)
-            if data.dragging then
-                self.frame:StopMovingOrSizing()
+        function self.callbacks.OnDragStop(frame)
```

Comment:
The logic is not really the problem. 
I just have no idea if a frame anchored to the hovered element would survive a scroll action
and how to create appropriate a frame/texture.